components/Card.js

```diff
@@ -1,7 +1,44 @@
 import React from "react";
+import Card from "react-bootstrap/Card";
 
-const Card = () => {
-	return <div>Card</div>;
+const Block = ({ month, day, year, type, time, place, description }) => {
+	return (
+		// have one shadow white and one black right behind it looking like a border
+
+		<Card
+			style={{
+				width: "15rem",
+				backgroundColor: "black",
+				borderRadius: 25,
+				boxShadow: "12px -13px white, 16px -17px black",
+			}}
+		>
+			<Card.Body>
+				<div className='flex justify-evenly '>
```

Comment:
Remove trailing whitespace

Reply:
I can't when I save it goes back to that